feature/networkinstance/networkinstance_test.go

```diff
@@ -85,6 +90,94 @@ func TestAugment(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		desc:     "Static route with no next-hops",
+		ni:       New("default", oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE).WithStaticRoute("1.1.1.1", []string{""}),
```

Comment:
How about WithStaticRoute("1.1.1.1", nil) ? Maybe that is a separate test case.

Reply:
Done!